.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Deployment
+on:
+    release:
+        types: [published]
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   name: Ballerina Build
+                uses: ballerina-platform/ballerina-action/@master
+                with:
+                    args:
+                        build -a -c
+                    env:
```

Comment:
Align `env` with `with`[L12], Otherwise this yaml give `Invalid Workflow File Error`

Reply:
Fixed